python/lsst/meas/deblender/baseline.py

```diff
@@ -1119,7 +1310,53 @@ def _overlap(xlo, xhi, xmin, xmax):
 
 
 def _handle_flux_at_edge(log, psffwhm, t1, tfoot, fp, maskedImage,
-                         x0, x1, y0, y1, psf, pk, sigma1, patchEdges):
+                         x0, x1, y0, y1, psf, pk, sigma1, patchEdges
+    ):
+    """
+    If a substantial amount of flux is found at the edge of the template,
+    this routine can be used to extend the template by the PSF and
+    "fill in" the footprint.
+
+    Parameters
+    ----------
+    log: `log.Log`
+        LSST logger for logging purposes.
+    psffwhm: `float`
+        PSF FWHM in pixels.
+    t1: `afw.image.ImageF`
+        The image template that contains the footprint to extend.
+    tfoot: `afw.detection.Footprint`
+        Symmetric Footprint to extend.
+    fp: `afw.detection.Footprint`
+        Parent Footprint that is being deblended.
+    maskedImage: `afw.image.MaskedImageF`
+        Full MaskedImage containing the parent footprint ``fp``.
+    x0,y0: `init`
+        Minimum x,y for the bounding box of the footprint ``fp``.
+    x1,y1: `int`
+        Maximum x,y for the bounding box of the footprint ``fp``.
+    psf: `afw.detection.Psf`
+        PSF of the image.
+    pk: `afw.detection.PeakRecord`
+        The peak within the Footprint whose footprint is being extended.
+    sigma1: `float`
+        Estimated noise level in the image.
+    patchEdges: `bool`
+        If ``patchEdges==True`` and if the footprint touches pixels with the
+        ``EDGE`` bit set, then for spans whose symmetric mirror are outside the
+        image, the symmetric footprint is grown to include them and their
+        pixel values are stored.
+    
+    Results
+    -------
+    t2: `afw.image.ImageF`
+        Image of the extended footprint.
+    tfoot2: `afw.detection.Footprint`
+        Extended Footprint.
```

Comment:
I was trying to be consistent , but sometimes referring to ``footprint`` is the same as referring to ``Footprint``, since ``footprint`` is a ``Footprint``. So I think I was consistent in the way I intended the names to be used, but I was apparently inconsistent in consistently letting the user know whether I really meant ``footprint`` or ``Footprint``. 😄